CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+* A **Meta CIP** describes a process surrounding Celo or proposes a change to (or an event in) a process. Process CIPs are like Standards Track CIPs but apply to areas other than the Celo protocol itself. They may propose an implementation, but not to Celo’s codebase; they often require community consensus; unlike Informational CIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include code of conduct updates, procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Celo development. Any Meta CIP is also considered a Process CIP.
+* An **Informational CIP** describes a Celo design issue, or provides general guidelines or information to the Celo community, but does not propose a new feature. Informational CIPs do not necessarily represent Celo community consensus or a recommendation, so users and implementers are free to ignore Informational CIPs or follow their advice.
+
+It is highly recommended that a single CIP contain a single key proposal or new idea.
+
+A CIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement as to be determined by the CIP editors during the PR review process. For more information on the CIP Editor role, see the appropriate section below.
+
+
+# CIP Workflow
+## Shepherding a CIP
+Parties involved in the process are you, the champion or _CIP author_, the CIP editors, and the Celo Core Developers (core developers are the developers working on the Celo client).
+
+Before you begin writing a formal CIP, you should vet your idea. Ask the Celo community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on Issues section of the CIP process to do this, but you can also use the Celo Discord and the [Celo Forum](https://forum.celo.org) and engage with the community about your idea.
+
+In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your CIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required to implement a Core CIP will be much greater than for a CRC and the CIP will need to be reviewed carefully by the Celo community. Negative community feedback will be taken into consideration and may prevent your CIP from moving past the Draft stage.
```

Comment:
We might need to clarify what kind of negative feedback might block a CIP, as I can imagine any significant proposal will have detractors. Especially in the case of CRCs, or other non-core designs, we would not want someone's politically motivated attacks to be able to block something the community wants. It may be something we hash out as we get experience in the process, but it may be worth talking about now, drawing on your experience from the Ethereum community.

Reply:
For CRCs, my opinion is that, since they don't really affect consensus but just a design specification, even if they receive negative feedback, it's still ok to implement them since they fall in the Ring 2-3 of Standards track. The only reason they shouldn't be accepted by editors is if they're not technically sound but clearly a CRC that isn't pushing the Celo Protocol forward and doesn't align with Celo Tenets and Community Code of Conduct.